pallets/rewards/src/mock.rs

```diff
@@ -94,6 +94,12 @@ pub enum CurrencyId {
 	M,
 }
 
+impl Default for CurrencyId {
+	fn default() -> Self {
+		CurrencyId::Reward
+	}
+}
```

Comment:
There is also a [default for enums](https://users.rust-lang.org/t/derive-default-for-enum-non-only-struct/44046). Anyways, no strong opinion here 👍🏻